.github/workflows/deploy.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
         with:
           name: deployment-build
           path: build/build.tgz
+          retention-days: 5
```

Comment:
is this something we should implement for all projects?

Reply:
Yeah I think so :)